### ISSUE TITLE
Please make password optional on Wi-fi

### ISSUE THREAD

**Issue by Manganga** [helium/hotspot-app#711]
I need to connect my bobcat miner to a public wifi that has no password. Please in the helium app have wifi password be optional. Please, my set up depends on it.

Thank you!

**Comment by cokes518** [comment]
We can try to remove the app validation and see if miner (on the Hotspot) accepts no password.

**Comment by Manganga** [comment]
Wow, you can give it a try for me?!

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by Manganga** [comment]
Can you please tell me when the password is off? I need to travel to reset the hotspot. So before I go, I'll like to know first when it's ready. Thank you so much!!

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by abhay** [comment]
> Can you please tell me when the password is off? I need to travel to reset the hotspot. So before I go, I'll like to know first when it's ready. 

Please follow the release notes. That will be your indication that the feature is released.

**Comment by Manganga** [comment]
Where are the release notes?

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by abhay** [comment]
In the App Store and on Discord.

**Comment by Manganga** [comment]
Thank you!

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by wolfenhawke** [comment]
Is this a good thing to do? By hosting an HS you are driving the network and taking responsibility for provisioning that area. Yes you are mining, but mining is secondary. Now if you use a public wi-fi to allow easier locating an HS, you are also opening up your HS for attacks. This seems like a trust reduction for that specific area/HS and maybe an easier intrusion into the network.

**Comment by wolfenhawke** [comment]
At a minimum, I would like the option to NOT relay via an open HS. Would not want my open ports available to an open HS.

**Comment by Manganga** [comment]
Is this being worked on now?

Sent from ProtonMail mobile

-------- Original Message --------

**Comment by cokes518** [comment]
no, it's under consideration. the team has lots of other work ahead of this.
